shepherd.js/src/components/shepherd-element.svelte

```diff
@@ -73,6 +78,41 @@
   function getClassesArray(classes) {
     return classes.split(' ').filter((className) => !!className.length);
   }
+  /**
+   * Handle tab focus within the dialog and the target element
+   *
+   * @private
+   */
+  function handleFocus(e) {
+    if (e.keyCode === KEY_TAB) {
+      if (focusableElements.length === 0) {
+        e.preventDefault();
+      }
+      // Backward tab
+      if (e.shiftKey) {
+        if (
+          document.activeElement === firstFocusableElement ||
+          document.activeElement.classList.contains('shepherd-element')
+        ) {
+          e.preventDefault();
+          lastFocusableElement.focus();
+        }
+      } else {
+        if (document.activeElement === lastFocusableElement) {
+          e.preventDefault();
+          // Focus target element when tabbing forward from last element
+          if (step.target) {
```

Comment:
I am not sure step.target is what we want here. I believe that is usually `document.body`. I think we will have to grab the element from `attachTo`.